src/netaddress.cpp

```diff
@@ -578,15 +615,6 @@ std::string CNetAddr::ToStringIP() const
     case NET_IPV4:
         return IPv4ToString(m_addr);
     case NET_IPV6: {
-        CService serv(*this, 0);
-        struct sockaddr_storage sockaddr;
-        socklen_t socklen = sizeof(sockaddr);
-        if (serv.GetSockAddr((struct sockaddr*)&sockaddr, &socklen)) {
-            char name[1025] = "";
-            if (!getnameinfo((const struct sockaddr*)&sockaddr, socklen, name,
```

Comment:
Was removing link-local scoped id functionality an intended consequence of this PR?

Reply:
https://github.com/bitcoin/bitcoin/issues/21982#issuecomment-842325937

Reply:
If we want to restore that functionality we could pass `m_scope_id` to `IPv6ToString` and append it if non-zero.